### ISSUE TITLE
Pass in --max-http-header-size for Node.js 8.15.0

### ISSUE THREAD

**Issue by ker-an** [Financial-Times/n-heroku-tools#552]
https://github.com/Financial-Times/n-heroku-tools/blob/0ada2df0ff875a7f3fdc81a743359193557e4193/tasks/run.js#L83

https://github.com/Financial-Times/n-heroku-tools/blob/0ada2df0ff875a7f3fdc81a743359193557e4193/tasks/run.js#L85

When running locally with version 8.15.0 or 10.15.1 and above, we should pass in an option when starting node.

```shell
node --max-http-header-size=80000
```

**Comment by ker-an** [comment]
What we've learned:

- Clustering isn't an issue. When we ran `node --max-http-header-size=80000 server/proxy.js` and `n-test smoke -H http://localhost:5050`, the smoke test passed locally

- Running the app directly works. When we ran `node --max-http-header-size=80000 server/cluster.js` and `n-test smoke -H http://localhost:5050`, the smoke test passed locally

**However** when we run `make run`, the smoke test times out. The error we get is:

`TimeoutError: Navigation Timeout Exceeded: 60000ms exceeded`

`make run` runs `nht run --script server/cluster.js` under the hood - see https://github.com/Financial-Times/n-heroku-tools/blob/master/tasks/run.js

Why does `nht run` not work?

**Comment by ker-an** [comment]
I added the following to [n-heroku-tools](https://github.com/Financial-Times/n-heroku-tools/blob/master/tasks/run.js):

```javascript
args.unshift('--max-http-header-size=80000');
```

It friggin' worked - the smoke test passed and is no longer timing out! When we tried the same solution yesterday, it didn't work. I'm not sure what changed.

To make sure my test wasn't a false positive, I purposely tried to break it. I made --max-http-header-size less than the header size in the smoke test (see below). It failed which leads me to believe that adding the line of code above works!

```javascript
// Just below 8500 header size in the smoke test
args.unshift('--max-http-header-size=8400'); 
```
